frontend/packages/core/src/AppLayout/drawer.tsx

```diff
@@ -1,198 +1,239 @@
+      backgroundColor: "#D7DAF6",
+    },
+  },
+  "&:hover, &:active, &.Mui-selected": {
+    ".MuiAvatar-root": {
+      backgroundColor: "#3548D4",
+    },
+    ".heading": {
+      color: "#3548D4",
+    },
+    ".initials": {
+      color: "#FFFFFF",
+    },
+  },
+  "&.Mui-selected": {
+    backgroundColor: "#FFFFFF",
+  },
+});
+
+const GroupHeading = styled(Typography)({
+  color: "rgba(13, 16, 48, 0.6)",
+  fontWeight: 500,
+  fontSize: "14px",
+  lineHeight: "18px",
+  flexGrow: 1,
+  paddingTop: "11px",
+});
+
+const Avatar = styled(MuiAvatar)({
+  background: "rgba(13, 16, 48, 0.6)",
+  height: "24px",
+  width: "24px",
+});
+
+const Initials = styled(Typography)({
+  color: "#FFFFFF",
+  fontSize: "14px",
+});
+
+// sidebar submenu
+const Popper = styled(MuiPopper)({
+  zIndex: 1200,
+  paddingTop: "16px",
+});
+
+const Paper = styled(MuiPaper)({
+  width: "230px",
+  border: "1px solid #E7E7EA",
+  boxShadow: "0px 10px 24px rgba(35, 48, 143, 0.3)",
+});
+
+// sudebar submenu groupings
+const LinkListItem = styled(ListItem)({
+  backgroundColor: "#FFFFFF",
+  height: "48px",
+  "&:hover": {
+    backgroundColor: "#F5F6FD",
+    "&.Mui-selected": {
+      backgroundColor: "#F5F6FD",
+    },
+  },
+  "&:active": {
+    backgroundColor: "#D7DAF6",
+    "&.Mui-selected": {
+      backgroundColor: "#D7DAF6",
+    },
+  },
+  "&:hover, &:active, &.Mui-selected": {
+    ".MuiTypography-root": {
+      color: "#3548D4",
+    },
+  },
+  "&.Mui-selected": {
+    backgroundColor: "#FFFFFF",
+  },
+});
 
-const Divider = styled(MuiDivider)`
-  ${({ theme }) => `
-  background-color: ${theme.palette.accent.main};
-  `}
-`;
+const LinkHeading = styled(Typography)({
+  color: "rgba(13, 16, 48, 0.6)",
+  fontWeight: 500,
+  fontSize: "14px",
+  lineHeight: "18px",
+});
 
 interface GroupProps {
   heading: string;
   open: boolean;
   updateOpenGroup: (heading: string) => void;
-  onNavigate: () => void;
 }
 
-const Group: React.FC<GroupProps> = ({
-  heading,
-  open = false,
-  updateOpenGroup,
-  onNavigate,
-  children,
-}) => {
-  const childrenList = React.Children.toArray(children);
+const Group: React.FC<GroupProps> = ({ heading, open = false, updateOpenGroup, children }) => {
+  const [openList, setListOpen] = React.useState(false);
+
+  const anchorRef = React.useRef(null);
+
+  const handleToggle = () => {
+    setListOpen(!openList);
+  };
+
+  const handleClose = event => {
+    if (anchorRef.current && anchorRef.current.contains(event.target)) {
+      return;
+    }
+    setListOpen(false);
+  };
+
+  let headingPath = window.location.pathname.replace("/", "").split("/")[0];
+
+  // TODO: discover this from config
+  if (headingPath === "ec2") {
+    headingPath = "aws";
+  }
+
+  const isSelected = headingPath == heading.toLowerCase();
 
   // n.b. if a Workflow Grouping has no workflows in it don't display it even if
   // it's not explicitly marked as hidden.
-  if (childrenList.length === 0) {
+  if (children === 0) {
```

Comment:
I think you want https://reactjs.org/docs/react-api.html#reactchildrencount
```suggestion
  if (React.Children.count(children) === 0) {
```